client/src/main/java/io/pravega/client/stream/notifications/notifier/SegmentNotifier.java

```diff
@@ -54,9 +54,9 @@ private void checkAndTriggerSegmentNotification() {
         int newNumberOfSegments = state.getNumberOfSegments();
         checkState(newNumberOfSegments > 0, "Number of segments cannot be zero");
 
-        if (this.numberOfSegments == 0) { //initialize the number of segments.
-            this.numberOfSegments = newNumberOfSegments;
-        } else if (this.numberOfSegments != newNumberOfSegments) { // SegmentNotification has happened.
+        //Trigger a notification with the initial number of segments, subsequent notifications are triggered only if there
```

Comment:
Could you break the line for this comment?

Reply:
done.